src/stratis_cli/_error_reporting.py

```diff
@@ -125,6 +126,10 @@ def _interpret_errors(errors):
                 "the D-Bus: %s."
             )
             return fmt_str % error
+
+        if isinstance(error, StratisCliUserError):
+            fmt_str = "Invalid user input: %s"
+            return fmt_str % error
```

Comment:
The ```isinstance``` checks for all subclasses of ```StratisCliInUseError``` become unreachable when you check ```isinstance(error, StratisCliInUseError)``` first. Since they can never be reached, you should delete them.

But, you should consider whether some part of the error messages that you see here in, e.g., StratisCliInUseError match block should be made part of the ```StratisCliInUseError```'s ```__str__``` method. You'll want to consider very carefully, so that the error messages will not become less readable.